cmds/sos/sos_test.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"sync"
+	"testing"
+)
+
+type RegistryEntryStub struct {
+	service string
+	port    uint
+}
+
+var (
+	knownServ1 = RegistryEntryStub{"stub1", 1}
```

Comment:
Do we have to worry about privileged system ports? Should there be a a test for that?

Reply:
This will be used on NiChrome, which is a one-user system. So the user is also the root user in this case.

Reply:
priveleged ports were never a good idea, and still are not, so I"m hoping we can dispense with that dstuff.

Reply:
From my friend Kevin Fall: "they provide no real security."
I think that's been the consensus for 35 years now ... we just can'' kill the damned things off. But priveleged ports are a myth.